snaps_k8s/common/consts/consts.py

```diff
@@ -95,8 +94,7 @@
 KUBESPRAY_FOLDER_NAME = 'kubespray'
 PROJ_DIR_NAME = 'snaps-k8s-projects'
 
-# TODO/FIXME - Use node config users
-NODE_USER = 'root'
+DFLT_NODE_USER = 'root'
```

Comment:
Change it as well if it makes sense.

Reply:
'root' is there just to keep the default behavior the same